applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.c

```diff
@@ -184,101 +236,172 @@ static void mcs_local_player_instance_cb(struct media_player *player, int err)
 	(void)media_proxy_ctrl_send_command(local_player, &cmd);
 }
 
-int ble_mcs_discover(struct bt_conn *conn)
+/**
+ * @brief	Send play or pause command to either local media player or peer
+ *
+ * param	cmd	Command to send
+ *
+ * @return	0 for success, error otherwise.
+ */
+static int play_pause(struct mpl_cmd *cmd)
+{
+	int ret = 0;
+
+	if (IS_ENABLED(CONFIG_BT_MCS)) {
+		ret = media_proxy_ctrl_send_command(local_player, cmd);
+	} else if (IS_ENABLED(CONFIG_BT_MCC)) {
+		/* Send cmd to all peers connected and has finished discovery */
+		for (uint8_t i = 0; i < CONFIG_BT_MAX_CONN; i++) {
+			if (mcc_peer[i].mcp_mcs_disc_status == FINISHED) {
+				if (mcc_peer[i].conn != NULL) {
+					ret = bt_mcc_send_cmd(mcc_peer[i].conn, cmd);
+					if (ret) {
+						LOG_WRN("Failed to send play/pause command: %d",
+							ret);
+						return ret;
+					}
+				}
+			} else {
+				LOG_WRN("MCS discovery has not finished: %d",
+					mcc_peer[i].mcp_mcs_disc_status);
+				return -EBUSY;
+			}
+		}
+	}
+
+	if (ret) {
```

Comment:
This can be moved to inside if (IS_ENABLED(CONFIG_BT_MCS))

Reply:
Can no longer see what this comment is about since it is outdated, so it might be fixed already